### ISSUE TITLE
Add authentication 

### ISSUE THREAD

**Issue by belldandu** [Voxel9/VitaDL#1]
I have a server which requires standard http authentication. When I attempt to dl a file from it using your vpk it crashes.

**Comment by Voxel9** [comment]
Yeah, admittedly this is quite a quick and dirty attempt at a simple file downloader (first attempt at Vita homebrew and what-not) and I never really gave authentication much thought at the time.
I'm slowly crawling back to the Vita scene, so if I get the time, I shall try to dedicate some time to fixing a lot of issues with his homebrew :)

-Voxel
